tests/scripts/pulp_python/test_sync.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+# shellcheck source=tests/scripts/config.source
+. "$(dirname "$(dirname "$(realpath "$0")")")"/config.source
+
+pulp debug has-plugin --name "pulp_python" --min-version "3.1.0.dev" || exit 3
+
+cleanup() {
+  pulp python remote destroy --name "cli_test_python_remote" || true
+  pulp python repository destroy --name "cli_test_python_repository" || true
+}
+trap cleanup EXIT
+
+cleanup
+
+# Prepare
+expect_succ pulp python remote create --name "cli_test_python_remote" --url "$PYTHON_REMOTE_URL" \
+  --includes '["shelf-reader", "aiohttp>=3.2.0,<3.3.1", "celery~=4.0", "Django>1.10.0,<=2.0.6"]'
```

Comment:
If you tend to judge projects by their dependencies, this one looks super bizarre. :exploding_head: